src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
CSS: You have not provided a fallback font in the `font-family` property. It's always a good idea to add a fallback font in case the primary font fails to load or is not supported by the user's browser. You should add a generic font family at the end of the list, like `serif` or `sans-serif`.